src/Robo/Plugin/Commands/DeployCommands.php

```diff
@@ -73,5 +84,47 @@ public function deployDrupal(string $appDirPath, string $siteName = 'default', s
             // https://github.com/drush-ops/drush/issues/2449#issuecomment-708655673
             ->exec("$appDirPath/vendor/bin/drush config:import --yes")
             ->run();
+        // @todo Do we want/need this option if we can just check for environment variables?
+        // @todo This could be useful if a site wanted to opt out of this functionality.
+        // @todo Perhaps we should rename this option then to be more generic?
+        if ($opts['tugboat']) {
+            $this->notifySlackOnFailedBasePreviewBuild($result);
+        }
+        return $result;
+    }
+
+    /**
+     * Notify Slack if a base preview build failed.
+     *
+     * @param \Robo\Result $result
+     *   The result of the task to check.
+     *
+     * @see https://docs.tugboatqa.com/starter-configs/code-snippets/slack-integration/
+     */
+    protected function notifySlackOnFailedBasePreviewBuild(Result $result): void
+    {
+        // Confirm we are in a Tugboat environment.
+        if (getenv('TUGBOAT_PREVIEW_ID') === false) {
```

Comment:
> Usher is using what Robo uses. PSR-12? I will look it up to verify, but we chose not to use Drupal coding standards on Usher since this is not a Drupal project.

Got it. I have taken note and will remember.